.env.sample

```diff
@@ -6,3 +6,7 @@ SND_SERVER_PORT=6660
 
 SND_SQLITE_DB_PATH=///./data.sqlite
 SND_RECOMMENDER_ARCHIVE_PATH=./recommender_model.zip
+
+OTEL_EXPORTER_OTLP_ENDPOINT="https://api.honeycomb.io"
+OTEL_EXPORTER_OTLP_HEADERS=<x-honeycomb-team=your-api-key>
+OTEL_SERVICE_NAME="soundations"
```

Comment:
Do we need to add those in `docker-compose.yml` as well so it works on prod server env?

Reply:
Sure, just forgot :)